packages/cli/tests/e2e/function/TabAddFunctionSql.tests.ts

```diff
@@ -41,30 +37,26 @@ describe("Configuration successfully changed when with different plugins", funct
     await CliHelper.addResourceToProject(projectPath, Resource.AzureSql);
 
     // Provision
-    setSimpleAuthSkuNameToB1Bicep(projectPath, environmentManager.getDefaultEnvName());
-    setBotSkuNameToB1Bicep(projectPath, environmentManager.getDefaultEnvName());
+    await setSimpleAuthSkuNameToB1Bicep(projectPath, environmentManager.getDefaultEnvName());
+    await setBotSkuNameToB1Bicep(projectPath, environmentManager.getDefaultEnvName());
     await CliHelper.setSubscription(subscription, projectPath);
-    await CliHelper.provisionProject(projectPath);
+    await CliHelper.provisionProject(
+      projectPath,
+      " --sql-admin-name Abc123321 --sql-password Cab232332"
```

Comment:
better not hard code password, even in test cases.

Reply:
Will fix in next PR to avoid dismissal of code owner's review approval. Thx